libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -348,6 +352,10 @@ export class ComboboxComponent
     /** @hidden */
     clearInputBtnFocused = false;
 
+    /** Can be used to add class to respective element */
+    addOnClass = '';
+    buttonClass = '';
```

Comment:
is it a public api? then we have to write typedocs for each property, method, otherwise `@hidden` and `_` prefis are needed. because we have tool that generates documentation from the comments in the code.

Reply:
yeah, it's public api. Did the changes.

Reply:
@shrvr is there a reason we want this to be public? If all of this should happen under the hood, it would be better to hide this from the API. And using `@Hostbinding` would be more succinct than using public properties and the `_applyShellbarMode` functions